system-test/busybench-js/src/busybench.js

```diff
@@ -15,7 +15,9 @@
  */
 
 const fs = require('fs');
+// eslint-disable-next-line node/no-extraneous-require
```

Comment:
Does gts v3 mis-identify this as an extraneous require?

Reply:
Yeah, it did. I was also confused, but this is how I made the linter pass.

Reply:
Yes; My best guess is that this happens because "pify" is a function rather than a package. But this was required to make this pass.